mindsdb/integrations/handlers/cohere_handler/requirements.txt

```diff
@@ -0,0 +1 @@
+cohere
```

Comment:
Let's pin this to `< 5.0`, otherwise this could silently break at some point if the cohere team releases a new major version.

Reply:
Resolved the comment, thanks for reviewing